config/locales/pages/fr.yml

```diff
@@ -4,3 +4,6 @@ fr:
     page_not_found:
       body: Vous pouvez revérifier votre lien et essayer de nouveau. (404)
       header: La page que vous recherchez n'existe pas.
+    page_took_too_long:
+      body: Vous pouvez vouloir attendre quelques minutes et réessayer. (503)
```

Comment:
Can we change to: `S'il vous plait, vous pouvez vouloir attendre duex or trois minutes et réessayer.`

Reply:
I think this can be simplified to `Veuillez réessayer dans quelques minutes`. `Veuillez` is a nice way of asking, so the `S'il vous plait` can be omitted. You'll notice that we don't use `S'il vous plait` anywhere in the app.

Also, I don't think we should be specifying a number of minutes because we don't know how long it will take for the app to start working again.

Reply:
ok: `Veuillez réessayer dans quelques minutes` it is. :)